integration-tests/grpc-external-proto/src/main/resources/dir/my-proto.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 option java_package = "com.test";
 
+option java_generic_services = true;
```

Comment:
Could you create a duplicated proto file?

Reply:
done